docs/documentation/functions/licenses/New-VCFLicenseKey.md

```diff
@@ -44,7 +44,7 @@ Accept wildcard characters: False
 
 ### -productType
 
-Specifies the product type for the license key. One of: SDDC_MANAGER, VCENTER, VSAN, ESXI, NSXT.
+Specifies the product type for the license key. One of: SDDC_MANAGER, VCENTER, VSAN, ESXI, NSXT, NSXV, WCP, HORIZON_VIEW.

```

Comment:
Are NSXV & HORIZON_VIEW still valid in 4.x/5/x?

Reply:
Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation/pull/226/commits/f6245c72edb6e807bf420610c77d00c6023c1d65